<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>662</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MeshSuite</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0" rowspan="6">
     <widget class="QFrame" name="viewerFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="glLayout"/>
       </item>
       <item row="0" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="6" column="0" rowspan="2">
     <widget class="QFrame" name="controlFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="infoControl">
         <property name="text">
          <string>Control</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="scaleLabel">
           <property name="text">
            <string>Zoom     </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="scaleSpinBox">
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="minimum">
            <double>0.000001000000000</double>
           </property>
           <property name="maximum">
            <double>1000000005.999999046325684</double>
           </property>
           <property name="singleStep">
            <double>0.500000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="openCenterLavel">
           <property name="text">
            <string>Center (</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="xTranslateDoubleSpinBox">
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="minimum">
            <double>-999999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="commaCenterLabel">
           <property name="text">
            <string>,</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="yTranslateDoubleSpinBox">
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="minimum">
            <double>-999999999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999999.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="closeCenterLabel">
           <property name="text">
            <string>)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="selectionTriangleLabel">
           <property name="text">
            <string>Selección Triángulo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="manualRadioButton">
           <property name="text">
            <string>Manual</string>
           </property>
           <property name="shortcut">
            <string>Ctrl+M</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="automaticRadioButton">
           <property name="text">
            <string>Automático</string>
           </property>
           <property name="shortcut">
            <string>Ctrl+A</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="stopButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>||</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="refineOnceButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&gt;</string>
           </property>
           <property name="shortcut">
            <string>Ctrl+Shift+Space</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="refineButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&gt;&gt;</string>
           </property>
           <property name="shortcut">
            <string>Ctrl+Space</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="4">
     <widget class="QFrame" name="optionFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QToolBox" name="optionToolBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <property name="tabSpacing">
          <number>1</number>
         </property>
         <widget class="QWidget" name="triangleSelectPage">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>437</width>
            <height>202</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Priorización de Triángulo para Refinar</string>
          </attribute>
          <layout class="QFormLayout" name="formLayout_5">
           <item row="4" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Parámetro de calidad</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QDoubleSpinBox" name="automaticTriangleSelectionDoubleSpinBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="value">
              <double>20.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Priorizar por:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QCheckBox" name="repeatSelectedTriangleCheckBox">
             <property name="text">
              <string>Repetir triángulo hasta destruir</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="automaticTriangleSelectionComboBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <item>
              <property name="text">
               <string>Sin Priorizar</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Ángulo Interno Menor</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Arista Más Pequeña</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Arista Más Pequeña, Prioridad Bordes</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Menor Circunradio</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Largo Borde</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="newPointPage">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>437</width>
            <height>202</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Criterio Selección Nuevo Punto para Insertar</string>
          </attribute>
          <layout class="QFormLayout" name="formLayout_3">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Método para Nuevo Punto:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QComboBox" name="newPointMethodComboBox">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <item>
              <property name="text">
               <string>Lepp - Bisección</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Circuncentro - Delaunay</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Lepp - Punto Medio - Delaunay</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Lepp - Centroide</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Off-Center - Delaunay</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Lepp - Circuncentro - Delaunay</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="insertionPage">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>437</width>
            <height>202</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Método Inserción de Nuevo Punto</string>
          </attribute>
          <layout class="QFormLayout" name="formLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="1">
            <widget class="QComboBox" name="insideNewPointCaseComboBox">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <item>
              <property name="text">
               <string>Flip Diagonal (Delaunay)</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="1" rowspan="4">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextEdit" name="infoTextEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1118</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuMesh">
    <property name="title">
     <string>PSLG</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionReload"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMesh"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>Nueva</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Abrir...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Cargar...</string>
   </property>
   <property name="toolTip">
    <string>Cargar Malla</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Recargar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Guardar como...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Acerca...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>refineOnceButton</tabstop>
  <tabstop>stopButton</tabstop>
  <tabstop>refineButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>on_refineOnce_clicked()</slot>
 </slots>
</ui>
